queue_services/entity-filer/tests/unit/test_worker/test_correction_bcia.py

```diff
@@ -213,6 +213,26 @@
     }
 }
 
+BC_CORRECTION_SHORT = {
+    'filing': {
+        'header': {
+            'name': 'correction',
+            'date': '2025-01-01',
+            'certifiedBy': 'system'
+        },
+        'business': {
+            'identifier': 'BC1234567',
+            'legalType': 'BC'
+        },
+        'correction': {
+            'details': 'First correction',
+            'correctedFilingId': '123456',
+            'correctedFilingType': 'incorporationApplication',
+            'comment': 'Correction for Incorporation Application filed on 2025-01-01 by system'
+        }
+    }
+}
```

Comment:
can we add even one small element to in this correction filing json so we can see that it actually is able to process changes in the correction json?